docs/index.md

```diff
@@ -22,20 +22,20 @@ Install the Fixie SDK using pip:
 $ pip install fixieai
 ```
 
-Set your `FIXIE_API_KEY` environment variable to your API key, which
-you can find in the Fixie web interface:
-```shell
-$ export FIXIE_API_KEY=<YOUR API KEY>
-```
-
+Once installed, run `fixie auth` to authenticate with Fixie and start building.
```

Comment:
I think we learned we need to tell them to make sure they've first authenticated and created an account on Fixie. Then they can auth